ddtrace/tracer/log.go

```diff
@@ -81,31 +78,29 @@ func logStartup(t *tracer) {
 	}
 
 	info := startupInfo{
-		Date:                        time.Now().Format(time.RFC3339),
-		OSName:                      osName(),
-		OSVersion:                   osVersion(),
-		Version:                     version.Tag,
-		Lang:                        "Go",
-		LangVersion:                 runtime.Version(),
-		Env:                         t.config.env,
-		Service:                     t.config.serviceName,
-		AgentURL:                    t.config.transport.endpoint(),
-		Debug:                       t.config.debug,
-		AnalyticsEnabled:            !math.IsNaN(globalconfig.AnalyticsRate()),
-		SampleRate:                  fmt.Sprintf("%f", t.rulesSampling.globalRate),
-		SamplingRules:               t.rulesSampling.rules,
-		ServiceMappings:             t.config.serviceMappings,
-		Tags:                        tags,
-		RuntimeMetricsEnabled:       t.config.runtimeMetrics,
-		HealthMetricsEnabled:        t.config.runtimeMetrics,
-		ApplicationVersion:          t.config.version,
-		ProfilerCodeHotspotsEnabled: t.config.profilerHotspots,
-		ProfilerEndpointsEnabled:    t.config.profilerEndpoints,
```

Comment:
Looks like `ProfilerCodeHotspotsEnabled` and `ProfilerEndpointsEnabled` were removed from here. Is that intentional?

Reply:
No, it wasn't intentional. I'm not quite sure how that happened... I'll add them back in, thanks for catching that